library/Ivoz/Provider/Domain/Model/BillableCall/BillableCallRepository.php

```diff
@@ -61,32 +62,31 @@ public function resetPricingData(array $ids);
     public function resetInvoiceId(int $invoiceId);
 
     /**
-     * @param array $conditions
-     * @param int $invoiceId
+     * @param InvoiceInterface $invoice
      * @return void
      */
-    public function setInvoiceId(array $conditions, int $invoiceId);
+    public function setInvoiceId(InvoiceInterface $invoice);
 
     /**
-     * @param int $companyId
-     * @param int $brandId
-     * @param string $startTime
-     * @param string $endTime
+     * @param InvoiceInterface $invoice
+     * @return mixed
+     */
+    public function getGeneratorByInvoice(InvoiceInterface $invoice);
+
+    /**
+     * @param InvoiceInterface $invoice
      * @return array
      */
-    public function getUntarificattedCallIdsInRange(int $companyId, int $brandId, string $startTime, string $endTime): array;
+    public function getUntarificattedCallIdsByInvoice(InvoiceInterface $invoice): array;
```

Comment:
Not related to this PR but I dont think _tarificated_ has two T's

Reply:
Replaced by unrated. Thanks :)